src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/DetectPreviewFeatureTests.Fields.cs

```diff
@@ -134,8 +134,8 @@ public class PreviewType
 
             }
 
-            public class {|#3:AGenericClass|}<T>
-                where T : PreviewType
+            public class AGenericClass<T>
+                where T : {|#3:PreviewType|}
```

Comment:
I don't expect this to work if it became a qualified name, alias qualified name, or nullable syntax.

See https://github.com/dotnet/roslyn-analyzers/pull/5425/files#r700017876

Reply:
Nice suggestion on nullable syntax elsewhere! I fixed that and modified some of the unit tests here to test the nullable syntax nodes